src/language/rule/rule.ts

```diff
@@ -79,10 +79,10 @@ export class RuleFailure {
     private ruleName: string;
 
     constructor(sourceFile: ts.SourceFile,
-                start: number,
-                end: number,
-                failure: string,
-                ruleName: string) {
+        start: number,
```

Comment:
assuming this is part of the indent rule? I like the previous alignment better, but I also like automation over manual work :)